test/e2e/tests/test_external_oidc.py

```
@@ -108,7 +348,147 @@ def test_minted_api_key_can_list_models_and_infer(self, maas_api_base_url: str):
+        Authorino's OIDC verification.
+        """
+        try:
+            tenant_b_url = _tenant_b_token_url()
+        except AssertionError:
+            pytest.skip("OIDC_ISSUER_URL does not contain 'tenant-a'; cannot derive tenant-b URL")
+
+        # Get a valid token from tenant-b
+        try:
+            tenant_b_token = _request_oidc_token(
+                username="charlie_sec_lead",
+                password="letmein",
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "password"

(S106)